docs/source/users/output/support_pages.rst

```diff
@@ -0,0 +1,34 @@
+.. _support_pages:
+
+=============
+Support Pages
+=============
+
+The support pages can be used to see more information about the problem and a
+given fit.
+
+Each page represents a single fitting combination, for example:
+
+- *Problem*: ``cubic`` from the ``simple_tests`` dataset
+- *Software*: ``scipy_ls``
+- *Minimizer*: ``lm-scipy``
+- *Jacobian*: ``scipy``
+- *Jacobian method*: ``2-point``
+
```

Comment:
he intention here was to highlight that a fitting combination covers all of these rather than say just the minimizer+dervatives. I'm happy to remove it and just leave
`Each page represents a single fitting combination and is split into 2 sections.`